monoids/README.md

```diff
@@ -45,4 +45,10 @@ program[
   ]
 ]|
 open program
-```
\ No newline at end of file
+```
+
+Reduces to a final value:
+
+```
+string[left[hello[]]|right[world[]]]
```

Comment:
The example reduces to `string[left[string[hello[]]]|right[string[world[]]]]`

Reply:
ah yes, you're right! 👍